doc/htmldoc/related_projects.rst

```diff
@@ -41,6 +41,13 @@ The app enables the rapid construction, parametrization, and instrumentation of
 
 * :doc:`Get started with NEST desktop <desktop:user/index>`
 
+NEST GPU
+--------
+
+NEST GPU is a GPU-MPI library for simulation of large-scale networks of spiking neurons. Iti can be used in Python, in C++ and in C.
```

Comment:
```suggestion
NEST GPU is a GPU-MPI library for simulation of large-scale networks of spiking neurons. It can be used in Python, in C++ and in C.
```